ui/index.d.ts

```diff
@@ -1113,6 +1113,8 @@ export interface IStorageAccountWizardContext extends IResourceGroupWizardContex
     storageAccount?: StorageManagementModels.StorageAccount;
 
     newStorageAccountName?: string;
+
+    isStack?: boolean;
```

Comment:
Could you add a comment describing what this will be used for?

Reply:
I add some comments, could you please take a look if they are good to you